Gloss.py

```diff
@@ -0,0 +1,62 @@
+import datetime
+
+class GlossEntry:
+	def __init__(self, name, acronym_full="", definition="", furtherreading="", institute="", internal=False, pronunciation="", seealso="", updated=datetime.date.today()):
+		self.name = name
+		self.acronym_full = acronym_full
+		self.definition = definition
+		self.furtherreading = furtherreading
+		self.institute = institute
+		self.internal = internal
+		self.pronunciation = pronunciation
+		self.seealso = seealso
+		self.updated = updated
+		# name (string) - entry's name, make sure to use correct capitalization/lack thereof
+		# acronym_full (string) - if acronym, what is the full name. if blank, assumed to not be an acronym
+		# further_reading (string) - URL to a webpage, usually an "official" one associated with the term
+		# institute (string) - optional, which institution is the phrase associated with, if any? For instance, Terra is associated with the Broad Institute. GCP is associated with Google. Does not necessarily imply a trademark or copyright.
+		# internal (bool) - Is this only used internally? For acronyms this may imply the acronym is internal but the phrase it refers to is not (ex: gha - github app)
+		# pronunciation (string) - optional pronunciation (ex: wdl - "widdle")
+		# updated (date) - when the entry was last updated
+
+
+	def generate_plaintext(self):
+		with open("output.txt", "a") as f:
+			f.write("\n\n")
+
+			# print name and underline it
+			f.write(f"{self.name}\n")
+			for x in range(0,len(self.name)):
+				f.write("-")
+			f.write("\n")
+
+			# print pronunciation
+			if self.pronunciation is not "":
```

Comment:
You don't want to use `is` for a comparison here. For a good explanation, check out this article: https://adamj.eu/tech/2020/01/21/why-does-python-3-8-syntaxwarning-for-is-literal/

Using is here should probably generate a syntax warning (depending on your version of Python). Using a good IDE could also help catch this mistake.

This comment applies to many of the other `is`s in your code. Let me know if you have further questions about this, I'm happy to clarify.